index.html

```diff
@@ -2,14 +2,54 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./styles.css">
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Quiz App</title>
+  <link rel="stylesheet" href="styles.css">
 </head>
 
 <body>
-    <script src="./script.js"></script>
+  <section class="players-container" id="players-container">
+    <label class="player1-label">Player 1:
+      <input type="text" class="p1-input" id="p1-input" placeholder="Enter Players 1 name"></label>
+    <label class="player1-label">Player 2:
+      <input type="text" class="p2-input" id="p2-input" placeholder="Enter Players 2 name"></label>
+    <button id="start-btn" class="start-btn" disabled>Start quiz!</button>
+    <div class="score" id="score"></div>
+    <button class="restart-btn" id="restart-btn">Restart!</button>
```

Comment:
I would prefer if the style setting `display: none` was written here in the HTML code, instead of the CSS file. It makes it clearer to the reader that this button is not meant to be shown when the page loads.